src/amp-tags.js

```diff
@@ -41,11 +41,12 @@ export function StoryAmpTags(seoConfig, config, pageType, data = {}, opts) {
   const { currentHostUrl = '', domainSlug } = data;
   // TODO: Remove this condition and always make absolute URL if that's better for AMP discoverability.
   const ampUrlAppend = seoConfig.appendHostToAmpUrl ? getDomain(currentHostUrl, domainSlug) || config['sketches-host'] : '';
+  const storySlug = seoConfig.decodeAmpUrl ? decodeURIComponent(story.slug): encodeURIComponent(story.slug);
```

Comment:
Can you please document this parameter?

Reply:
Otherwise it looks ok

Reply:
tried to add doc for it, but still throwing 404 in malibu doc.